src/kouros/kouros.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"errors"
+	"kouros/ha"
+	"kouros/pos"
+)
+
+
+
+func NewPOSManager(managerType string) (pos.POSManager, error) {
```

Comment:
Is it okay to have a string managerType? How about using const or enum variables?

Reply:
This makes sense. We will change it to enum variables